simtools/io_handler.py

```diff
@@ -85,16 +87,14 @@ def get_output_directory(self, label=None, dir_type=None, test=False):
         if self.use_plain_output_path:
             path = Path(self.output_path)
         else:
-            if test:
-                output_directory_prefix = Path(self.output_path).joinpath("test-output")
-            else:
-                output_directory_prefix = Path(self.output_path).joinpath("simtools-output")
-
-            today = datetime.date.today()
-            label_dir = label if label is not None else "d-" + str(today)
+            output_directory_prefix = Path(self.output_path).joinpath(dir_type[:8] + "-output")
```

Comment:
This construct allows to handle the special case of `dir_type = simtools-results` without too many if / else statements. Any better idea to do that efficiently?

What is missing is to handle the 'None' case, otherwise there is no danger in terms of length of arrays:

```
>>> dir_type="a"
>>> print(dir_type[:8])
a
>>> dir_type=None
>>> print(dir_type[:8])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not subscriptable
```